lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -0,0 +1,247 @@
+;(function (define, undefined) {
+'use strict';
+define(['jquery', 'underscore', 'annotator_1.2.9'], function ($, _, Annotator) {
+    /**
+     * Adds the Accessibility Plugin
+     **/
+    Annotator.Plugin.Accessibility = function () {
+        _.bindAll(this,
+            'addAriaAttributes', 'onHighlightKeyDown', 'onViewerKeyDown',
+            'onEditorKeyDown'
+        );
+        // Call the Annotator.Plugin constructor this sets up the element and
+        // options properties.
+        Annotator.Plugin.apply(this, arguments);
+    };
+
+    $.extend(Annotator.Plugin.Accessibility.prototype, new Annotator.Plugin(), {
+        pluginInit: function () {
+            this.annotator.subscribe('annotationViewerTextField', this.addAriaAttributes);
+            this.annotator.element.on('keydown.accessibility.hl', '.annotator-hl', this.onHighlightKeyDown);
+            this.annotator.element.on('keydown.accessibility.viewer', '.annotator-viewer', this.onViewerKeyDown);
+            this.annotator.element.on('keydown.accessibility.editor', '.annotator-editor', this.onEditorKeyDown);
+            this.addTabIndex();
+        },
+
+        destroy: function () {
+            this.annotator.unsubscribe('annotationViewerTextField', this.addAriaAttributes);
+            this.annotator.element.off('keydown.accessibility.hl');
+            this.annotator.element.off('keydown.accessibility.viewer');
+            this.annotator.element.off('keydown.accessibility.editor');
```

Comment:
Can we just use `this.annotator.element.off('.accessibility');` to unbind all keydown events here?